src/client/java/teammates/client/scripts/util/LoopHelper.java

```diff
@@ -0,0 +1,39 @@
+package teammates.client.scripts.util;
+
+/**
+ * Utility class to be used with loops to print messages regularly.
+ */
+public class LoopHelper {
```

Comment:
Now that you have narrowed the scope of this class to just printing, you may want to name the class to fit the scope. `LoopHelper` is too general.